### DISCUSSION TITLE
`Invert` transform for MetaTensor

### DISCUSSION THREAD

**Discussion by surajpaib** [Project-MONAI/MONAI#7289] [General]
**Is your feature request related to a problem? Please describe.**
The existing [`Invertd`](https://docs.monai.io/en/latest/transforms.html#invertd) transform provides useful functionality to invert transforms applied during training. I was not able to, however find an `Invert` transform that works only on MetaTensor

**Describe the solution you'd like**
I think it would be beneficial to have this feature for MetaTensor so it can be used outside dict formats. 

**Describe alternatives you've considered**
Our current workaround is to just iterate over the transforms and call their `inverse` property in custom code. 

**Additional context**

**Comment by KumoLiu** [comment]
Hi @surajpaib, thanks for your reporting.
I think we have a "vanilla" one for `Invert`.
https://github.com/Project-MONAI/MONAI/blob/88f8dd275dbe8ca856b45ca34d6d01f4ea4c0128/monai/transforms/post/array.py#L831
Thanks!

**Comment by ibro45** [comment]
I also wasn't aware it existed; seems like it's missing in the docs https://docs.monai.io/en/latest/transforms.html
